wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsnext/model/ChePluginReference.java

```diff
@@ -70,7 +70,7 @@ public boolean equals(java.lang.Object o) {
     if (o == null || getClass() != o.getClass()) {
       return false;
     }
-    CheServiceReference cheServiceReference = (CheServiceReference) o;
+    ChePluginReference cheServiceReference = (ChePluginReference) o;
```

Comment:
cheServiceReference -> chePluginReference
Please make sure that all usages of `service` is replaced with `plugin`

Reply:
I'm not going to update Feature related model as we are going to remove this entity from the model according to the discussion in workspace next walking skeleton https://github.com/eclipse/che/issues/10123

Reply:
OK, but it's not about feature model changing, but about variable name =)

Reply:
done